src/main/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiService.scala

```diff
@@ -99,6 +99,26 @@ trait WorkspaceApiService extends HttpService with PerRequestCreator with UserIn
             WorkspaceService.UnlockWorkspace(WorkspaceName(workspaceNamespace, workspaceName)))
         }
       }
+    } ~
+    path("workspaces" / Segment / Segment / "invites" ) { (workspaceNamespace, workspaceName) =>
+      post {
+        requireUserInfo() { userInfo =>
+          entity(as[WorkspaceACLUpdate]) { invite =>
```

Comment:
update: i'm going to go with #3 because per the http spec, delete technically shouldn't be allowed to have a json body